src/cloud/release-notes/mcp-release-notes.md

```diff
@@ -17,30 +17,36 @@ The `{{site.data.var.mcp}}` package uses the following version sequence: `<major
 See [Apply patches]({{site.baseurl}}/cloud/project/project-patch.html) to learn how to apply Magento patches and hot fixes to your {{site.data.var.ece}} project.
 
 ## v1.0.2
-*Release date: February 24, 2020*<br/>
+*Release date: February 27, 2020*<br/>
 
--  {:.fix}**Added the [PayPal Express Checkout issue with region patch for Magento v2.3.4](https://magento.com/tech-resources/download#download2353)**-Published on February 12, 2020,  this patch resolves an issue that affects orders placed with PayPal Express Checkout where the shipping address for the order specifies a country region that has been manually entered into the text field rather than selected from the drop-down menu on the Shipping page. See [Apply patches]({{site.baseurl}}/cloud/project/project-patch.html).
+This release includes the following patches and critical fixes:
 
--  {:.fix}**Updated the [Scope parameter for Async/Bulk API patch](https://magento.com/tech-resources/download#download2312) for Magento Open Source v2.3.1 and 2.3.2**-Fixed a syntax error in the `composer.json` file.
+-  {:.new}**Compatibility updates for `{{site.data.var.mcp}}`**–
+
+   -  Updated the `symfony` and `semver` version constraints in the `composer.json` file for compatibility with Magento 2.4 and later releases.<!--MAGECLOUD-5127-->
+
+   -  Updated constraints in `composer.json` for compatibility with `{{site.data.var.ct}}` 2002.0.22 and later 2002.0.x releases.
 
--  {:.fix}**Compatibility fixes to support upcoming Magento releases**–Updated the `symfony` and `semver` version constraints in the `composer.json` file for compatibility with Magento 2.4 and later.<!--MAGECLOUD-5127-->
+-  {:.new}**Added the [PayPal Express Checkout issue with region patch for Magento v2.3.4](https://magento.com/tech-resources/download#download2353)**-Published on February 12, 2020,  this patch resolves an issue that affects orders placed with PayPal Express Checkout where the shipping address for the order specifies a country region that has been manually entered into the text field rather than selected from the drop-down menu on the Shipping page. See [Apply patches]({{site.baseurl}}/cloud/project/project-patch.html).
 
--  {:.fix}**Compatibility fix to support the `{{site.data.var.ct}}` package**–Updated constraints in `composer.json` for compatibility with `{{site.data.var.ct}}` 2002.0.22 and later 2002.0.x releases.
+-  {:.new}Added a patch to fix an issue that disabled the full page cache during the Magento deployment process. This patch applies to Magento 2.3.2 and later releases.
+
+-  {:.fix}**Updated the [Scope parameter for Async/Bulk API patch](https://magento.com/tech-resources/download#download2312) for Magento Open Source v2.3.1 and 2.3.2**-Fixed a syntax error in the `composer.json` file.
```

Comment:
Also, a long time ago we had a problem with the environment variables and using a link for the bolded part. It looked strange to bold a link. I think we decided the compromise was to add a statement that made it clear where the link would take the reader. I think that would apply here, too. The link should come at the end of the release statement and be clear about where it leads.